tests/e2e-openshift/multi-cluster/create_otlp_sender.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Get the HTTP and GRPC routes from OpenTelemetry receiver collector.
+otlp_route_http=$(oc -n kuttl-multi-cluster-receive get route otlp-http-otlp-receiver-route -o json | jq '.spec.host' -r)
+otlp_route_grpc=$(oc -n kuttl-multi-cluster-receive get route otlp-http-otlp-receiver-route -o json | jq '.spec.host' -r)
+
+# Define the template content
+template_content=$(cat <<EOF
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: otel-collector-template
+parameters:
+- name: OTLP_ROUTE_HTTP
+  description: HTTP route
+  required: true
+- name: OTLP_ROUTE_GRPC
+  description: GRPC route
+  required: true
+objects:
+- apiVersion: opentelemetry.io/v1alpha1
+  kind: OpenTelemetryCollector
+  metadata:
+    name: otel-sender
+    namespace: kuttl-multi-cluster-send
+  spec:
+    mode: deployment
+    image: docker.io/otel/opentelemetry-collector-contrib:latest
```

Comment:
I did not check how we make this in other tests, but could it lead to issues that we use latest?

Reply:
I think we can remove it, I was specifying it to try out basicauth and bearer token auth. Let me check and update the test case.

Reply:
I need to check how routes can be used on Kind clusters.

Reply:
We require the collector image from 'contrib' because the 'resourcedetection' processor is not part of the core. Utilizing the latest tag will enable us to detect any regressions early, especially if there are changes to any of the 'contrib' features. Regarding the routing aspect, I couldn't find any references to it except within the 'e2e-openshift' testsuite. It might be beneficial to consider updating the upstream ingress test case to accommodate Kind clusters since Ingress is widely used in the upstream Kubernetes ecosystem.